src/chat/chat.controller.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Body,
+  Controller,
+  Post,
+  UseGuards,
+  BadRequestException,
+  NotFoundException,
+  Param,
+  Get,
+} from '@nestjs/common';
+import { ChatService } from './chat.service';
+import { CreateRoomDto } from './dto/create-room.dto';
+import { JwtAuthGuard } from '../common/guards/jwt.guard';
+import { ChatGateway } from './chat.gateway';
+import { ReqUser } from '../common/decorators/user.decorator';
+import { JwtPayloadDto } from 'src/common/payload/jwt-dto';
+
+@Controller('chat')
+export class ChatController {
+  constructor(
+    private readonly rooms: ChatService,
+    private readonly gateway: ChatGateway, // socket
```

Comment:
이부분도 [이전 PR 리뷰사항](https://github.com/jungyoubin/mini_project/pull/24#discussion_r2314686304)처럼 네이밍 변경하면 좋을 것 같습니다.
```suggestion
    private readonly chatService: ChatService,
    private readonly chatGateway: ChatGateway, // socket
```

Reply:
수정 완료 하였습니다. 감사합니다.